### PR TITLE
app-text/zathura-pdf-poppler: add 0.3.2

### PR DESCRIPTION
**Pull request by andrewpak**
Hello Gentoo!

I just bumped zathura-pdf-poppler, let me know if this is okay!

Thanks!!

### CODE REVIEW

File: app-text/zathura-pdf-poppler/zathura-pdf-poppler-0.3.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (Kangie) [inline app-text/zathura-pdf-poppler/zathura-pdf-poppler-0.3.2.ebuild:4]: EAPI=8

**Comment by Kangie** [comment]
Few minor QA issues that we can fix while we're here:

- update poppler dependency to >= 21.12
- The bugtracker has moved, please update `metadata.xml`
- The `http` URIs can be updated to `https`
- we can drop 0.3.1 (keep -r1)
- Copy the updated ebuild to -9999 to update the live template.

LGTM otherwise - zathura can successfully open PDFs!

**Comment by andrewpak** [comment]
* Updated the bugtracker in `metadata.xml`.
* `http` now `https`
* 0.3.1 dropped
* template copied to 9999

I'm not sure about updating the poppler dependency since it doesn't seem to be a dependency of this package directly?

**Comment by Kangie** [comment]
> I'm not sure about updating the poppler dependency since it doesn't seem to be a dependency of this package directly?

We'll be fine as all versions in ::gentoo are newer than the dependency, but it's nice that the meson build system included a good lower bound :). The trick here is the `poppler-glib` is called `app-text/poppler` on gentoo :)

**Comment by andrewpak** [comment]
Okay, I think I got the header line back. Let me know if there's anything else that needs to fix, when I get home later tonight I'll look more into the poppler dep version.

**Comment by andrewpak** [comment]
Alright, everything should be good now, I updated the poppler dependency to 23.12.0. I had to change poppler[cairo] to just poppler, as when I was trying to test with 23.12.0 and having [cairo] there it said "invalid atom". Not sure if that matters but testing the build was fine on my side.

**Comment by Kangie** [comment]
In this case, the [cairo] actually is important, as it instructs portage to make sure that poppler is built with support for the cairo rendering library.

It's just a syntax thing; try this instead: `>=app-text/poppler-21.12.0:=[cairo]` :)

Remember to apply it to the live ebuild as well, and if doing it in a separate commit, try a commit message like:

`app-text/zathura-pdf-poppler: update poppler dependency`

On the whole though, looking good!

**Comment by andrewpak** [comment]
My bad, I stepped out after my last commit. I amended the commit and set the poppler dep >=23.12.0 (thats the current stable version).  Everything should be good now. Thanks Kangie, I appreciate your help!

**Comment by andrewpak** [comment]
The poppler dependency is now >=21.12.0 now as you suggested in the case that older systems try to update. Thanks again for taking the time to review this and giving me some guidance, much respect!

**Comment by juippis** [comment]
@Kangie all yours :)

**Comment by andrewpak** [comment]
I see what's wrong now. Had to step out, I'll fix it when I get back tonight.

**Pull request closed without merging**